packages/cli/src/commands/start.ts

```diff
@@ -188,12 +188,19 @@ export class Start extends BaseCommand {
 		const license = Container.get(License);
 		await license.init(this.instanceId);
 
+		const hasCert = (await license.loadCertStr()).length > 0;
```

Comment:
These changes look unrelated to the PR title. can we create a separate PR for this change please 🙏🏽